src/System.Text.Json/tests/Serialization/CyclicTests.cs

```diff
@@ -19,14 +19,28 @@ public static void WriteCyclicFail()
         }
 
         [Fact]
-        [ActiveIssue(37313)]
         public static void WriteTestClassWithArrayOfElementsOfTheSameClassWithoutCyclesDoesNotFail()
         {
             TestClassWithArrayOfElementsOfTheSameClass obj = new TestClassWithArrayOfElementsOfTheSameClass();
 
             //It shouldn't throw when there is no real cycle reference, and just empty object is created
             string json = JsonSerializer.ToString(obj);
-            Assert.Equal(@"{}", json);
+            Assert.Equal(@"{""Array"":null}", json);
```

Comment:
@rynowak previously suggested that this return an empty object (i.e. `{}`):
https://github.com/dotnet/corefx/pull/37611#discussion_r284524201

Reply:
The array is null, so it has a null value like every other null property.